toolbox

```diff
@@ -70,6 +70,7 @@ toolbox_image=""
 toolbox_runtime_directory="$XDG_RUNTIME_DIR"/toolbox
 user_id_real=$(id -ru 2>&3)
 verbose=false
+podman="podman"
```

Comment:
Nice.

Nitpick: There's no need to describe that you added a `podman` variable to the code in the commit's change log, because it's clearly visible in the `diff`. The change log is mainly useful for things that are not immediately obvious from looking at the code changes. Things like why the change was made, what other approaches were considered and what the trade-offs were, any weird nuances of the code, etc..

Reply:
Thanks for the comment. I'll rework the message a bit.